### DISCUSSION TITLE
求助，链接线label样式设置

### DISCUSSION THREAD

**Discussion by yangshangbo** [antvis/X6#2137] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42314632/170404224-1a6144b2-e762-49a0-b739-c2a490874636.png)
请问上面这种label样式如何实现？

下面是我实现出来的效果，没有边距设置导致看起来很拥挤
![image](https://user-images.githubusercontent.com/42314632/170404525-e12c79a5-b07f-42c2-95e7-1c81a152cfc7.png)
我的label配置项：
```javascript
{
      attrs: {
        label: {
          text: label,
        },
        rect: {
          fill: '#ffffff',
          stroke: 'rgba(0, 0, 0, 0.25)',
          rx: 10,
          ry: 10,
          refHeight: 4,
          refX: 0,
          refY: -2,
        },
      },
}
```

**Comment by NewByVector** [comment, marked as answer]
希望这个 [demo](https://codesandbox.io/s/she-zhi-biao-qian-de-margin-xiao-guo-ylw06n) 能帮到你。

**Reply by yangshangbo** [reply]
非常感谢，是我需要的效果。
